docs/developer_documentation/ref-python.md

```diff
@@ -0,0 +1,226 @@
+``` 
+Now you can create a python script. 
+To get started, you should import the python api, connect to a molgenis server and login:
+```python
+import molgenis
+session = molgenis.Session.("https://molgenis.mydomain.example/api/")
+session.login("username","password")
+```
+Always put the import and molgenis.Session in your script to make the api work. 
+
+# Overview example
+```python
+import molgenis
+session = molgenis.Session.("https://molgenis.mydomain.example/api/")
+session.login("username","password")
+my_table = session.get("package_entityName")
+print(my_table)
+```
+To get a full overview of all functions in the python api, download the following zip:
+[MolgenisPythonFunctions.zip](../data/MolgenisPythonFunctions.zip)
+# Methods
+## login
+```python
+session.login(username,password)
+```
+Login to the MOLGENIS REST API<br/><br/>
+
+Argument    | Description                                       | Required | Default
+------------|---------------------------------------------------|----------|--------
+`username`  | Username for a registered molgenis user           | yes      |
+`password`  | Password for the user                             | yes      | 
+
+## logout
+```python
+session.logout()
+```
+Logout from the MOLGENIS REST API and destroy the session.
+
+
+##getById
+```python
+session.getById("tableId", "rowId")
+```
+Retrieves a single entity row from an entity repository.<br/><br/>
+
+Argument    | Description                                       | Required | Default
+------------|---------------------------------------------------|----------|--------
+`entity`    | Fully qualified name of the entity                | yes      |
+`id`        | The value for the idAttribute of the entity       | yes      | 
+`attributes`|  The list of attributes to retrieve               | no       | All attributes
+`expand`    | The attributes to expand                          | no       | None
+
+
+##get
+```python
+session.get("package_entityName")
+```
+
+Retrieves entities and returns the result in a dataframe.
+
+Argument    | Description                                          | Required | Default
+------------|------------------------------------------------------|----------|--------
+`entity`    | Fully qualified name of the entity                   | yes      |
+`q`         | Query string in rsql/fiql format (see below)         | no       | None
+`start`	    | The index of the first row to retrieve (zero indexed)| no       | 0
+`num`       | The maximum number of rows to return (max 10000)     | no       | 100
+`attributes`| The list of attributes to retrieve                   | no       | All attributes
+`sortColumn`| attributeName of the column to sort on               | no       | None
+`sortOrder` | The order to sort in                                 | no       | None
+
+
+**Supported RSQL/FIQL query operators (see [https://github.com/jirutka/rsql-parser](https://github.com/jirutka/rsql-parser))**
```

Comment:
This is not entirely true, we have our own subset of this guy
But we do not have documentation for this yet.

I guess this will do